### ISSUE TITLE
Allow specifying the layout on render

### ISSUE THREAD

**Issue by danielkhan** [fastify/point-of-view#276]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

Currently, it seems as if a layout can only be specified when registering a template engine while it should be an option when rendering individual views.
Registering a property for each layout is a workaround but this duplicates a lot of code during runtime, allocating memory.

### Motivation

The sole purpose of layouts is that they can be chosen per view render. When I render the view, I know which layout to use.

### Example

```js
  fastify.register(pov, {
    engine: { ejs },
    root: config.viewdir,
    layoutRoot: config.layoutdir, // Optionally specify where to find all layouts
    includeViewExtension: true,
    defaultContext: {
      dev: process.env.NODE_ENV === 'development',
    },
  });
  
  // ...

  fastify.get('/', async (request, reply) => {
    reply.view('someview', {}, 'somelayout');
  });
```

**Comment by mcollina** [comment]
Good idea! 
Would you like to send a Pull Request to address this issue? Remember to add unit tests.

**Comment by danielkhan** [comment]
Closed via #284
